src/mapeo-manager.js

```diff
@@ -658,13 +657,18 @@ export class MapeoManager extends TypedEmitter {
         )
       }
 
+      this.#activeProjects.set(projectPublicId, project)
+
       // 5. Wait for initial project sync
       if (waitForSync) {
-        await this.#waitForInitialSync(project)
+        try {
+          await this.#waitForInitialSync(project)
+        } catch (e) {
+          this.#l.log('ERROR: could not do initial project sync', e)
+        }
       }
-
-      this.#activeProjects.set(projectPublicId, project)
     } catch (e) {
+      // Only happens if getProject fails
```

Comment:
I think maybe change this try...catch to only wrap the getProject call, since everything else is wrapped in a fallthrough try...catch it seems now to make it hard to follow having this parent try...catch wrapping everything.